### PR TITLE
[ISSUE #1714] Add test case for shenyu-plugin-sign.

### PR DESCRIPTION
**Pull request by liyeping**
Complete #1714.

Make sure that: shenyu-plugin-sign

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor/).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-integrated-test/shenyu-integrated-test-http/src/test/java/org/apache/shenyu/integratedtest/http/SignTest.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import org.apache.shenyu.integratedtest.http.dto.UserDTO;
+import org.apache.shenyu.integratedtest.http.helper.HttpHelper;
+import org.junit.Test;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.util.Map;
+import java.util.concurrent.Future;
+import static org.junit.Assert.assertEquals;
+
+public class SignTest extends AbstractTest{
```
Comment (dengliming) [inline shenyu-integrated-test/shenyu-integrated-test-http/src/test/java/org/apache/shenyu/integratedtest/http/SignTest.java:32]: Can you reformat the code style?
```suggestion
public class SignTest extends AbstractTest {
```
The same as other places. Thanks.

Comment (liyeping) [reply shenyu-integrated-test/shenyu-integrated-test-http/src/test/java/org/apache/shenyu/integratedtest/http/SignTest.java]: I have formatted the code style, please review it again.

**Review by flycash — DISMISSED** [review]
I think the integrated tests looks fine

File: shenyu-integrated-test/shenyu-integrated-test-http/src/test/java/org/apache/shenyu/integratedtest/http/helper/HttpHelper.java [hunk 2]
```diff
@@ -52,4 +55,16 @@
         String respBody = response.body().string();
         return gson.fromJson(respBody, type);
     }
+
+    public <RESP> RESP getFromGateway(String path, Map<String, Object> headers, Type type) throws IOException {
+        Request.Builder requestBuilder = new Request.Builder().url(GATEWAY_END_POINT + path);
+        if(!CollectionUtils.isEmpty(headers)) {
```
Comment (dengliming) [inline shenyu-integrated-test/shenyu-integrated-test-http/src/test/java/org/apache/shenyu/integratedtest/http/helper/HttpHelper.java:61]:
```suggestion
        if (!CollectionUtils.isEmpty(headers)) {
```

File: shenyu-integrated-test/shenyu-integrated-test-http/src/test/java/org/apache/shenyu/integratedtest/http/helper/HttpHelper.java [hunk 3]
```diff
@@ -52,4 +55,16 @@
         String respBody = response.body().string();
         return gson.fromJson(respBody, type);
```
Comment (dengliming) [inline shenyu-integrated-test/shenyu-integrated-test-http/src/test/java/org/apache/shenyu/integratedtest/http/helper/HttpHelper.java:56]: `return this.getFromGateway(path, null, type);`

**Review by dengliming — CHANGES_REQUESTED** [review]

**Review by dengliming — APPROVED** [review]
@liyeping Thanks.

**Pull request merged**